apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -1639,7 +1729,8 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
         this.isGenDtlCollapsed = true;
         this.isMlngAddrCollapsed = true;
         this.isOthrDtlCollapsed = false;
-        this.forceClear$ = observableOf({status: true});
+        this.forceClear$ = observableOf({ status: true });
+        this.invoiceForceClearReactive$ = observableOf({ status: true });
```

Comment:
Add doc

Reply:
@rohit-walkover this is written in reset method and not member declaration